docs/product-assembly_business-optimizer-ER-sample-openshift/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,24 @@
+[id='assembly_optimizer-running-employee-rostering-openshift']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Running the employee roster sample for Business Optimizer
```

Comment:
Prakash strongly prefers to refer to this as a "template" instead of an "example" or "sample", because it's more than an average example. I am less bothered by this though.

Reply:
My suggestion is to use "starter application". The GA version does not look like a template, to be honest.